.github/workflows/run_all_steps.yml

```diff
@@ -16,7 +16,7 @@ on:
 
 env:
   DEBIAN_FRONTEND: noninteractive
-  SHADOW_COMMIT: 9eef66b4da7c78c31fa7d93c2982311a55f0abd4
+  SHADOW_COMMIT: 4d219cddb2f91932c2d6bf9af70328e8eb0e2c56
```

Comment:
+1 to wait for the 2.1 release to push, and then also update this commit to 2.1

Reply:
Done.